app/services/idv/usps_mail.rb

```diff
@@ -16,6 +17,12 @@ def any_mail_sent?
       user_mail_events.any?
     end
 
+    def most_recent_otp_expired?
+      return false unless any_mail_sent?
+
+      user_mail_events.first.updated_at < USPS_CONFIRMATION_WINDOW_DAYS.days.ago
```

Comment:
FWIW when I was thinking of implementing this, I would have put the `otp_created_at` in the PII payload next to the OTP itself in the encrypted PII. I don't think there's a clearly right answer here, but `updated_at` is a little ambiguous here for my comfort.

Maybe the `mail_spammed` should be using `created_at`? (since in cases we create the event for every attempt....updating shouldn't matter?) I don't feel as strongly about that since it's for rate limiting and I expect that to be more fluid compared to a really clear expiration point for the code.